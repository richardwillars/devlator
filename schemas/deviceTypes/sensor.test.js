const sensorModule = require("./sensor");
const constants = require("../../constants");
const eventsModule = require("../events");

const events = eventsModule(constants);

describe("models/sensor", () => {
  it("should export the commands", () => {
    const sensorModel = sensorModule(events, constants);
    expect(sensorModel.commands).toMatchSnapshot();
  });

  it("should export the events", () => {
    const sensorModel = sensorModule(events, constants);
    expect(sensorModel.events).toMatchSnapshot();
  });
});
