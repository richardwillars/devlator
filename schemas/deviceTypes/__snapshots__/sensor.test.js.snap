// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`models/sensor should export the commands 1`] = `Object {}`;

exports[`models/sensor should export the events 1`] = `
Array [
  "CONTACT",
  "TEMPERATURE",
  "HUMIDITY",
  "LIGHT",
  "UV",
  "BATTERY_LEVEL",
  "MOTION",
  "NAME",
  "VIBRATION",
  "TAMPER",
]
`;
